src/svg/path/tests/test_parsing.py

```diff
@@ -1,259 +1,593 @@
-import unittest
```

Comment:
Is there some sort of whitespace change?

Reply:
I don't know why it it didn't give a more proper change log. You could just copypasta the tests out of there,  or know that it only failed the one.

Reply:
OK, will do.